### PR TITLE
feat(ui): Make issue row gray when issue action

### PR DESCRIPTION
**Pull request by mikellykels**
Make the issue row gray when an issue action is performed (Resolve, Ignore), just like marking it as Mark Reviewed.

FIXES [WOR-802](https://getsentry.atlassian.net/browse/WOR-802)

**Marking as Resolved**

Screen Shot 2021-04-14 at 2 46 59 AM

**Marking as Ignored**

Screen Shot 2021-04-14 at 2 47 25 AM

**Dark Mode**
Screen Shot 2021-04-14 at 3 37 21 AM

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
What's this bad boy look like in dark mode?

File: src/sentry/static/sentry/app/components/stream/group.tsx [hunk 1]
```diff
@@ -86,6 +86,7 @@ type Props = {
 type State = {
   data: Group;
   reviewed: boolean;
+  issueAction: boolean;
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/stream/group.tsx:89]: how about `actionTaken`?

Comment (mikellykels) [reply src/sentry/static/sentry/app/components/stream/group.tsx]: ohh I like that - updated

**Comment by mikellykels** [comment]
updated the description of what it'll look like in dark mode :)

**Review by scttcper — APPROVED** [review]

**Pull request merged**